fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/event/LootTableLoadingCallback.java

```diff
@@ -33,17 +33,17 @@
 public interface LootTableLoadingCallback {
 	@FunctionalInterface
 	interface LootTableSetter {
-		void set(LootTable supplier);
+		void set(LootTable table);
 	}
 
 	Event<LootTableLoadingCallback> EVENT = EventFactory.createArrayBacked(
 			LootTableLoadingCallback.class,
-			(listeners) -> (resourceManager, manager, id, supplier, setter) -> {
+			(listeners) -> (resourceManager, manager, id, table, setter) -> {
 				for (LootTableLoadingCallback callback : listeners) {
-					callback.onLootTableLoading(resourceManager, manager, id, supplier, setter);
+					callback.onLootTableLoading(resourceManager, manager, id, table, setter);
 				}
 			}
 	);
 
-	void onLootTableLoading(ResourceManager resourceManager, LootManager manager, Identifier id, FabricLootSupplierBuilder supplier, LootTableSetter setter);
+	void onLootTableLoading(ResourceManager resourceManager, LootManager manager, Identifier id, FabricLootTableBuilder table, LootTableSetter setter);
```

Comment:
Might as well rename `LootManager manager` to `lootManager` (it was a leftover name from before the ResourceManager parameter was added).